CreeDictionary/API/schema.py

```diff
@@ -33,6 +33,14 @@ class SerializedWordform(TypedDict):
     definition: List[SerializedDefinition]
 
 
+class SerializedLinguisticTag(TypedDict):
+    # The tag in its original form, e.g., from the FST
+    value: str
+
+    # The value of the feature, written in plain English
+    in_plain_english: str
+
+
 class SerializedSearchResult(TypedDict):
     # the text of the match
     matched_cree: str
```

Comment:
Why do we make it explicit that this is Cree (rather than our non-English language)?